robocorp-code/vscode-client/src/extensionCreateEnv.ts

```diff
@@ -172,6 +182,67 @@ async function verifyLongPathSupportOnWindows(rccLocation: string): Promise<bool
     return true;
 }
 
+export async function ensureConvertBundle(justCheck: boolean = false) {
```

Comment:
Instead of a flag to `justCheck`, it should always do the check the first time it's called and update accordingly (but then, save in a global var if it had checked already and if everything is ok don't download anything anymore if the file is present locally and the previous check worked).

Reply:
This is because we want to update the bundle when a new one comes out (but we should just do the update check once per session, so, just recheck for a new version after a VSCode restart).